setup.cfg

```diff
@@ -35,7 +35,7 @@ py_modules = nbqa
 install_requires =
     ipython>=7.8.0
     tokenize-rt>=3.2.0
-    toml
+    tomli<2.0.0
```

Comment:
You may want a different version constraint, but this is what I'd personally choose. You'll find my thoughts about this here https://github.com/psf/black/pull/2301#discussion_r647748572